pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/api.go

```diff
@@ -172,6 +172,21 @@ func (s *apiScaffolder) scafffoldControllerWithImage(scaffold *machinery.Scaffol
 			return fmt.Errorf("error scaffolding container port in the controller: %v", err)
 		}
 	}
+
+	if len(s.runAsUser) > 0 {
```

Comment:
Also it is missing one part.
If we do not inform --run-as-user then we should not add this line at all.